src/blueapi/cli/scratch.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import os
+from pathlib import Path
+from subprocess import Popen
+
+from git import Repo
+
+from blueapi.config import ScratchConfig
+
+
+def setup_scratch(config: ScratchConfig) -> None:
+    _validate_directory(config.root)
+
+    logging.info(f"Setting up scratch area: {config.root}")
+
+    # Set umask to DLS standard
+    os.umask(0o002)
```

Comment:
Is the `o` meant to be here?

Reply:
Umask takes octal values, right? So this is octal 002, should it actually be `0o0002`?

Reply:
Yeah you're correct - I've just been so used to calling it from bash as a "normal" number that it surprised me.